src/cluster_legacy.h

```diff
@@ -305,6 +305,7 @@ struct _clusterNode {
     clusterLink *link;          /* TCP/IP link established toward this node */
     clusterLink *inbound_link;  /* TCP/IP link accepted from this node */
     list *fail_reports;         /* List of nodes signaling this as failing */
+    int node_health;            /* Last updated node health, can be stale. Update by calling updateNodesHealth() */
```

Comment:
it is still not very clear to me `node_health` is a boolean or not.

Reply:
It is a Boolean, will mention it in the comments

Reply:
Use a boolean name then, like `is_node_healthy`